peps/pep-0792.rst

```diff
@@ -0,0 +1,371 @@
+Post-History: `03-Feb-2025 <https://discuss.python.org/t/79356/>`__
+
+Abstract
+========
+
+This PEP proposes a standardized set of project status markers, as well
+as a mechanism for communicating those markers in the HTML and JSON
+simple indices.
+
+Rationale and Motivation
+========================
+
+The "status" of a project is an important piece of metadata, made more important
+by growth in both the breadth and depth of the Python packaging ecosystem.
+Both humans and tools use project statuses (or proxies for status,
+such as recent activity) to determine whether the project is maintained or
+otherwise suitable for consumption.
+
+Python packaging has at least three different mechanisms for communicating
+the "status" of a project:
+
+1. Distributions can include *trove classifiers* in their metadata, as
+   originally specified in :pep:`301`. The list of supported classifiers is
+   `maintained by the PyPA <https://github.com/pypa/trove-classifiers>`_,
+   and includes the ``Development Status`` hierarchy. For example, a
+   distribution can include the ``Development Status :: 7 - Inactive``
+   classifier to indicate that the distribution's project is inactive.
+
+   Trove classifiers are flexible, but also come with significant limitations:
+   they're machine-readable and are rendered on indices like PyPI, but
+   they also require the maintainer to push one or more *new* distributions
+   each time they wish to update their project's development status.
+   Furthermore, because distributions are *de facto* immutable in the Python
+   packaging ecosystem, older distributions can't have their classifiers
+   updated to reflect the current status of the project.
+
+2. Indices can mark distributions and releases as "yanked", as originally
+   specified in :pep:`592`. Yanked distributions are not considered
+   eligible for dependency resolution.
+
+   When a distribution has been yanked, it is marked with ``data-yanked``
+   in the HTML index and with ``yanked: bool | str`` in the JSON index.
+   Additionally, indices like PyPI will hide yanked distributions by default
+   and will render them with a notice when the user navigates directly to them.
+
+   Yanking is machine-readable like trove classifiers, but is single-purpose
+   rather than general-purpose: users can specify a free-form "yank reason,"
+   but the semantics of yanking are fixed.
+
+3. PyPI itself has *project statuses*, which apply to the entire project
+   (i.e., all releases and distributions). Project statuses have both
+   maintainer- and index-admin-controllable states:
+
+   * PyPI administrators can "quarantine" a project. Quarantine behaves like
+     a strengthened yank: the project remains entirely uninstallable while
+     quarantined.
+
+   * Project maintainers can "archive" a project. Archiving a project
+     disables new release and distribution uploads to that project,
+     but otherwise has no effect on the project's downloadability.
```

Comment:
[downloadability](https://books.google.com/ngrams/graph?content=downloadability%2Cdownload&year_start=1800&year_end=2022&corpus=en&smoothing=3) isn't a word (in common usage).
```suggestion
     but otherwise has no effect on the ability to download a project.
```